### ISSUE TITLE
Cannot get "installed_paths" to work

### ISSUE THREAD

**Issue by ghnp5** [squizlabs/PHP_CodeSniffer#2834]
Windows 10
Latest VSCode and Extensions

I have "slevomat/coding-standard" installed, and everything has been working well, until a few days ago (but not sure since when).

I noticed that VSCode wasn't reporting errors to me anymore, and now that I ran "phpcs", I get this error:

> .\vendor\bin\phpcs --standard=phpcs.xml --extensions=php .\src\ -v -p -s
> Registering sniffs in the MyStandard standard... ERROR: Referenced sniff "SlevomatCodingStandard.TypeHints.TypeHintDeclaration" does not exist
> 
> Run "phpcs --help" for usage information

(that's the first ref of SlevomatCodingStandard in phpcs.xml)

I had this in phpcs.xml:

`<config name="installed_paths" value="../../slevomat/coding-standard"/>`

To fix, I also tried:

`<config name="installed_paths" value="vendor/slevomat/coding-standard"/>`
(and tried with backslashes too)
`<config name="installed_paths" value="C:\........\vendor\slevomat\coding-standard"/>`

I also tried setting through the command line:

> $ vendor\bin\phpcs --config-set installed_paths vendor\slevomat\coding-standard
> Using config file: ..........\vendor\squizlabs\php_codesniffer\CodeSniffer.conf
> 
> Config value "installed_paths" added successfully
> 
> $ vendor\bin\phpcs --config-set installed_paths ..\..\slevomat\coding-standard
> Using config file: ..........\vendor\squizlabs\php_codesniffer\CodeSniffer.conf
> 
> Config value "installed_paths" updated successfully; old value was "vendor\slevomat\coding-standard"

But no luck.
If I completely remove "installed_paths" and all the references to SlevomatCodingStandard's rules, phpcs starts working again (with the built-in rules).

My setup:
1) "phpcs.bat" is located in "vendor\bin"
2) Built-in coding standards are located in "vendor\squizlabs\php_codesniffer\src\Standards"
3) SlevomatCodingStandard's standards are located in "vendor\slevomat\coding-standard"

What exactly do I have to do to include SlevomatCodingStandard's standards?...

Thank you.

**Comment by TracKer** [comment]
@ghnp5 How did you manage to fix this issue?

**Comment by ghnp5** [comment]
In `phpcs.xml`:

`<config name="installed_paths" value="../../slevomat/coding-standard,../../sirbrillig/phpcs-variable-analysis"/>`


Workspace settings:

```
		"phpSniffer.executablesFolder": "..\\myrepo\\vendor\\bin\\",
		"phpSniffer.run": "onSave",
		"phpSniffer.standard": "..\\myrepo\\phpcs.xml",
```

`composer.json`:

`"slevomat/coding-standard": "^8.10",`


Not sure if I'm missing anything else..
